test/Microsoft.Framework.Runtime.Tests/PathHelpers.cs

```diff
@@ -25,13 +25,7 @@ public static string GetRootedPath(params string[] paths)
                 return root;
             }
 
-            return Path.Combine(root, paths.Aggregate(Path.Combine));
-        }
-
-        private static bool IsWindows()
-        {
-            var p = (int)Environment.OSVersion.Platform;
-            return (p != 4) && (p != 6) && (p != 128);
+            return Path.Combine(root, paths.Aggregate((path1, path2) => Path.Combine(path1, path2)));
```

Comment:
Because CoreCLR only has Path.Combine(string []) and it needed Path.Combine(string, string)